src/components/modals/Send/Footer.js

```diff
@@ -0,0 +1,52 @@
+// @flow
+
+import React from 'react'
+
+import type { T, Account } from 'types/common'
+import type { DoubleVal } from 'components/RequestAmount'
+
+import { ModalFooter } from 'components/base/Modal'
+import Box from 'components/base/Box'
+import Label from 'components/base/Label'
+import FormattedVal from 'components/base/FormattedVal'
+import Button from 'components/base/Button'
+import Text from 'components/base/Text'
+
+type Props = {
+  t: T,
+  account: Account,
+  amount: DoubleVal,
+  onNext: Function,
+}
+
+function Footer({ account, amount, t, onNext }: Props) {
+  return (
+    <ModalFooter horizontal align="center">
+      <Box grow>
+        <Label>{t('send:totalSpent')}</Label>
+        <Box horizontal flow={2} align="center">
+          <FormattedVal
+            color="dark"
+            val={amount.left * 10 ** account.unit.magnitude}
```

Comment:
amount.left should be directly in the correct (sat) value. (also, it's better to have a `Math.floor` of this)

Reply:
basically idea is we should never have any float currencies, it's always normalized as an integer (in satoshis / cents). float should never exist in our data model (and only exists if we import data from some places that don't support integer style / format it back to a string)